src/index.html

```diff
@@ -8,6 +8,82 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars">
+      <div class="stars__star stars--0">
```

Comment:
modificator should be in `stars` BEM-block instead of each `stars__star` element. It should look like this:

```suggestion
    <div class="stars stars--0">
      <div class="stars__star">
```

Reply:
Then you should add changes to CSS (use chained selector - [CSS combinator](https://www.w3schools.com/css/css_combinators.asp))